tests/test_utils.py

```diff
@@ -124,34 +133,117 @@ def test_ribs_from_kisslinger_format():
         and poloidal angles expected.
       * The first and last ribs have the same R, Z data.
       * The first and second ribs do not have the same R, Z data.
+      * Data is formatted correctly.
     """
-    ribs_file = Path("files_for_tests") / "kisslinger_file_example.txt"
+    original_ribs_file = (
+        Path("files_for_tests") / "kisslinger_file_example.txt"
+    )
+    scrambled_ribs_file = (
+        Path("files_for_tests") / "kisslinger_file_scrambled.txt"
+    )
 
     (
-        custom_surface_toroidal_angles,
+        _,
+        _,
+        _,
+        _,
+        original_surface_coords,
+    ) = ribs_from_kisslinger_format(
+        original_ribs_file,
+        delimiter=" ",
+        scale=1,
+    )
```

Comment:
Since this is only used in a test near the end of this method, perhaps move this function call there.  Might also name the returned data as `expected_surface_coords`?